compose/compose-file/index.md

```diff
@@ -27,4 +27,5 @@ how to upgrade, see **[About versions and upgrading](compose-versioning.md)**.
 - [Installing Compose](../install.md)
 - [Compose file versions and upgrading](compose-versioning.md)
 - [Sample apps with Compose](../samples-for-compose.md)
+- [Enabling GPU access with Compose](../gpu-support-in-compose.md)
```

Comment:
Is this link correct?

Reply:
(the file that's added below is named "gpu-support.md"), but reading on my phone, os perhaps it was an existing file

Reply:
confirmed that it was broken; opened a PR to fix: https://github.com/docker/docker.github.io/pull/12323